docs/sql-ref-syntax-ddl-alter-database.md

```diff
@@ -18,5 +18,23 @@ license: |
   See the License for the specific language governing permissions and
   limitations under the License.
 ---
+### Description
+You can author metadata associated with a database by setting the `DBPROPERTIES`. Using this command 
```

Comment:
Remove "the" before `DBPROPERTIES` here and below?
And maybe back-tick SCHEMA and DATABASE.